### PR TITLE
candidate update from oldRound to newRound

### PR DESCRIPTION
**Pull request by nodeexcel**

### CODE REVIEW

File: src/controllers/fetchEmail.js [hunk 1]
```diff
@@ -148,6 +154,8 @@ export class FetchController extends BaseAPIController {
                         if (error) {
                             next(new Error(err));
                         } else {
+                            logsBody.logMsg = 'Unread status is successfully changed to Read';
```
Comment (reactexcel) [inline src/controllers/fetchEmail.js]: move such message to config file so i can easily edit them.

File: src/controllers/fetchEmail.js [hunk 2]
```diff
@@ -148,6 +154,8 @@ export class FetchController extends BaseAPIController {
...
+                            PubSub.PubSub.publish('changeUnreadStatus', logsBody);
```
Comment (reactexcel) [inline src/controllers/fetchEmail.js]: i don't want PubSub.PubSub. can you change import accordingly

File: src/controllers/pubSub.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+function userLogsFunction(body, userLog) {
+    return new Promise((resolve, reject) => {
+        userLog.create(body).then(insertData => {
+            resolve(insertData);
+        })
+    })
+}
+
+PubSub.subscribe('moveRounds', function (msg, data) {
```
Comment (reactexcel) [inline src/controllers/pubSub.js]: even these event names move it to the config file.

File: src/controllers/fetchEmail.js (hunk 1 above)
Comment (nodeexcel) [reply src/controllers/fetchEmail.js]: O.k Sir I will correct it.

File: src/controllers/fetchEmail.js (hunk 2 above)
Comment (nodeexcel) [reply src/controllers/fetchEmail.js]: O.k Sir I will correct it.

File: src/controllers/pubSub.js (hunk 3 above)
Comment (nodeexcel) [reply src/controllers/pubSub.js]: O.k, I will move it to config file.

File: config.json [hunk 4]
```diff
@@ -27,5 +27,33 @@
     "bitlyHost": "http://api.bit.ly/v3/shorten",
     "mongoConnectionString":"mongodb://127.0.0.1:27017/hr_recruit_dev",
     "loginRoleUrl": "http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/API_HR/api.php",
-    "loginEmailUrl": "http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/sal_info/api.php"
+    "loginEmailUrl": "http://176.9.137.77/hr_tms/ReactReduxHR/backend/attendance/sal_info/api.php",
```
Comment (reactexcel) [inline config.json]: this should be removed why did u add this again?

Comment (nodeexcel) [reply config.json]: Sir not added line again, only added comma ( , ) at last

**Pull request merged**